### ISSUE TITLE
Cannot submit form containing a field called "contains" in Unpoly 3

### ISSUE THREAD

**Issue by tomhillgreyridge** [unpoly/unpoly#507]
### Bug description

If you have a form with a field called "contains" in it, Unpoly appears unable to submit it, failing with "[element].contains is not a function" where [element] varies - is normally "root" if the form is in the top layer.

### Reproduction project
https://glitch.com/edit/#!/confirmed-periodic-form

### Steps to reproduce the behavior:

The first form contains a text field called "contains".  Clicking the submit button will raise a javascript error.
The second form contains a text field called "contains2".  Clicking the submit button works fine.

### Expected behavior

Both forms should be submittable as "contains" is a valid form field identifier

### Browser version
Chrome (latest) on Windows 10

**Comment by adam12** [comment]
Thank you for the reproduction. I've confirmed it here as well. Interesting find :)

**Comment by tomhillgreyridge** [comment]
Yeah.  It was so weird I assumed I had done something stupid!  

It looks like when the button is clicked Unpoly tries to work out whether the focused element is within the form by calling a method called contains to work out if it is in a subtree .  However, if the root is a form then form.[fieldname] gets you a reference to the input field so you get a conflict if the field name is contains

```javascript  
function isInSubtree(root, selectorOrElement) {
  const element = getOne(selectorOrElement);
  return root.contains(element);
}
```

I've added a "debug" button which demonstrates this

**Comment by triskweline** [comment]
The fix has been released as part of version 3.2.1.
